src/Locations.js

```diff
@@ -206,51 +210,66 @@ export default class Locations extends Component {
-                                                {this.state[location].map((item, index) => (
-                                                    <Draggable
-                                                        key={item}
-                                                        draggableId={item}
-                                                        index={index}>
-                                                        {(provided, snapshot) => (
-                                                            <div
-                                                                ref={provided.innerRef}
-                                                                {...provided.draggableProps}
-                                                                {...provided.dragHandleProps}
-                                                                style={getItemStyle(
-                                                                    snapshot.isDragging,
-                                                                    provided.draggableProps.style
-                                                                )}>
-                                                                {item.split('-').join(' ')}
-                                                            </div>
-                                                        )}    
-                                                    </Draggable>
-                                                ))}
-                                            </DroppableLocation>
-                                        ))}
-                                        </div>
+                                      <Tabs>
+                                        <TabList>
+                                          {Object.keys(LOCATIONS[locationGroup]).map((locationName) =>(
+                                            <Tab>{locationName}</Tab>
+                                          ))}
+                                        </TabList>
+                                          {Object.keys(LOCATIONS[locationGroup]).map((locationName) => (
+                                            <TabPanel>
+                                                  <div style={{
+                                                      display: 'flex',
+                                                      flexWrap: 'wrap',
+                                                      justifyContent: 'space-around',
+                                                      maxHeight: '75vh',
+                                                      overflowY: 'auto'
+                                                  }}>
+                                                  {Object.keys(LOCATIONS[locationGroup][locationName]).map((location) => (
+                                                    <DroppableLocation
+                                                      key={LOCATIONS[locationGroup][locationName][location]}
```

Comment:
`LOCATIONS[locationGroup][locationName][location]` should just be `location`

Reply:
This would cause the key to be a number instead of one of the names due to the way I changed the LOCATIONS constant. The current release to my understanding uses the names.

Reply:
Oh, I missed the `Object.keys` part. That's unnecessary - you can just directly map over the `LOCATIONS[locationGroup][locationName]` array.